### PR TITLE
Improved "Go To Bracket" to jump to next bracket.

### PR DESCRIPTION
**Pull request by PxlBuzzard**
Worked with @rebornix to implement an improved version of "Go To Bracket" that behaves more like Vim. Now when "Go to Bracket" is run and select is not on a bracket, it will go to the next bracket. The current behavior of swapping between the open and close bracket remains the same.

### CODE REVIEW

File: src/vs/editor/contrib/bracketMatching/common/bracketMatching.ts [hunk 1]
```diff
@@ -119,21 +119,28 @@ export class BracketMatchingController extends Disposable implements editorCommo
...
+			const nextBracket = model.findNextBracket(position);
+			if (nextBracket.range) {
+				resultingPosition = nextBracket.range.getStartPosition();
+			}
+		}
 
 		if (resultingPosition) {
 			this._editor.setPosition(resultingPosition);
 			this._editor.revealPosition(resultingPosition);
+		}
```
Comment (rebornix) [inline src/vs/editor/contrib/bracketMatching/common/bracketMatching.ts]: It seems this additional `}` breaks the compile.

**Comment by rebornix** [comment]
@PxlBuzzard it seems the additional `}` breaks build and leads to the CI failures.

**Comment by PxlBuzzard** [comment]
I just did some more testing and realized this doesn't handle multi-cursor yet. I will see if I can come up with a fix for that.

**Comment by rebornix** [comment]
LGTM, thanks!

**Pull request merged**